lib/dnsimple/client/registrar.rb

```diff
@@ -21,6 +21,14 @@ def register(account_id, domain_name, attributes = {}, options = {})
         Dnsimple::Response.new(response, Struct::Domain.new(response["data"]))
       end
 
+
+      def check(account_id, domain_name, attributes = {}, options = {})
+        options  = options.merge(attributes)
+        response = client.get(Client.versioned("/%s/registrar/domains/%s/availability" % [account_id, domain_name]), options)
```

Comment:
The correct path should be `"/%s/registrar/domains/%s/check"`. Did you get this path from somewhere? It may be an outdated docs.